metrics-facade-prometheus/src/main/java/com/ringcentral/platform/metrics/reporters/prometheus/PrometheusMetricsExporter.java

```diff
@@ -38,6 +38,10 @@ public enum Format {
         Format(String contentType) {
             this.contentType = contentType;
         }
+
+        public String getContentType() {
```

Comment:
We don't have a style guide yet, but if you don't mind, please remove the 'get' prefix so that the name aligns with the rest of the code.

Reply:
Fixed, thanks.